Makefile

```diff
@@ -26,7 +26,8 @@ widgets: \
 	www/widgets/lab9-browser.html www/widgets/lab9.js www/widgets/server9.js \
 	www/widgets/lab10-browser.html www/widgets/lab10.js www/widgets/server10.js \
 	www/widgets/lab11-browser.html www/widgets/lab11.js \
-	www/widgets/lab12-browser.html www/widgets/lab12.js
+	www/widgets/lab12-browser.html www/widgets/lab12.js \
+	www/widgets/lab12-browser.html www/widgets/lab13.js
```

Comment:
Uhh, is this a typo? `lab12-browser.html` appears twice!

Reply:
Fixed